openstack/nova/templates/prometheus-alerts.yaml

```diff
@@ -1,6 +1,6 @@
 {{- $values := .Values }}
 {{- if $values.alerts.enabled }}
-{{- range $target := list "kubernetes" "openstack" "metrics-metal" }}
+{{- range $target := list  "metrics-metal" "openstack" }}
```

Comment:
additional space.

why do you see a need for that change? commit-message of [[nova] update Prometheus Alert target folders](https://github.com/sapcc/helm-charts/pull/8678/commits/4b923c83307cc330a210d30b46e71010cc83166b) seems empty